ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1323,10 +1323,16 @@ literals:
   - ON
   - OFF
   
+
+#Addition of new fields for Tanks##
+- level_status  
```

Comment:
what enum states? list them.

Reply:
Added Low, Medium and High as the states in the telemetry